app/templates/account/prepaid-view.jade

```diff
@@ -30,16 +30,16 @@ block content
                 .form-group
```

Comment:
Oops, missed that in my testing. Good catch! Added c0172c434742d794e983b698d5cfa7c5b3ba9315